src/host/os_getversion.c

```diff
@@ -75,10 +75,14 @@ SYSTEM_INFO getsysteminfo()
 
 void getversion(struct OsVersionInfo* info)
 {
-	OSVERSIONINFOEX versionInfo = {0};
+	RTL_OSVERSIONINFOEXW versionInfo = {0};
 
-	versionInfo.dwOSVersionInfoSize = sizeof(OSVERSIONINFOEX);
-	GetVersionEx((OSVERSIONINFO*)&versionInfo);
+	versionInfo.dwOSVersionInfoSize = sizeof(RTL_OSVERSIONINFOEXW);
+
+	typedef LONG (WINAPI* RtlGetVersionSig)(PRTL_OSVERSIONINFOEXW);
+	RtlGetVersionSig getVersion = (RtlGetVersionSig)
+	GetProcAddress(GetModuleHandle(TEXT("ntdll.dll")), "RtlGetVersion");
+	if ( getVersion ) getVersion(&versionInfo);
```

Comment:
yikes, need to install the windows driver sdk for those. (they aren't included in the Win10 SDK at least)
And I highly doubt that's installed in a default environment...

imo the code is simple enough, but if you really want to force people to install that extra sdk just for simple version info's, be my guest...